apps/docs/ui.inc

```diff
@@ -0,0 +1,27 @@
+;;;;;;;;;;;;;;;;;;;;;;;;
+; user interface actions
+;;;;;;;;;;;;;;;;;;;;;;;;
+
+(import "././edit/ui.inc")
+
+;overload Edit app ui.inc
+(defun action-file-leaf-action ()
+	;load up the file selected
+	(defq file (. *file_tree* :get_route
+		(. *window* :find_id (getf *msg* +ev_msg_action_source_id))))
+	(select-node file)
+	(populate-page file))
+
+(defun action-close ()
+	(setq *running* :nil))
+
```

Comment:
does this not exist in edit/ui.inc ?

Reply:
It does but it relies on vdu which is not being used in the docs app.

Reply:
ok